core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -72,6 +72,40 @@ <h3>Send a Test Mail to Admin</h3>
   </div>
 </md-card>
 
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Flush the Memory Cache</h3>
+  <div class="oppia-query-form">
+    <form class="form" ng-submit="$ctrl.flushMemoryCache()">
+      <div class="form-group row">
+        <input type="submit" value="Flush Cache">
+      </div>
+    </form>
+    <form class="form" ng-submit="$ctrl.getMemoryCacheProfile()">
```

Comment:
Actually, one suggestion. Maybe don't have a button for this -- just show the results automatically on page/tab load?

(This is optional.)